test/integration/features/addon-registry-route.feature

```diff
@@ -0,0 +1,45 @@
+@addon-registry-route @addon
+Feature: registry-route add-on
+registry-route exposes private docker registry,
+so that user can push their images to private registry and delpoy it.
+
+  Scenario: As a user, I can enable the registry-route add-on
+    Given user starts shell instance on host machine
+     When executing "minishift addons enable registry-route" succeeds
+     Then stdout should contain "Add-on 'registry-route' enabled"
+
+  Scenario: As a user, I can starts Minishift
+    Given Minishift has state "Does Not Exist"
+      And image caching is disabled
+     When executing "minishift start --iso-url centos" succeeds
```

Comment:
@amitkrout We have documented this behavior. See https://docs.openshift.org/latest/minishift/troubleshooting/troubleshooting-misc.html#authentication-required-to-push-image

Reply:
yesterday @gbraad and @praveenkumar convey the same when i was using b2d. 

```docker push``` is successful in b2d iso if ```Docker CE``` is used in host irrespective of docker version inside the vm.  Now i am thinking can we add this to the documentation as a workaround ?cc_ @gbraad @praveenkumar